packages/telegram-cloud-storage-stamper/README.md

```diff
@@ -0,0 +1,77 @@
+# @turnkey/telegram-cloud-storage-stamper
+
+[![npm](https://img.shields.io/npm/v/@turnkey/telegram-cloud-storage-stamper?color=%234C48FF)](https://www.npmjs.com/package/@turnkey/telegram-cloud-storage-stamper)
+
+This package contains functions to store a turnkey api public/private key within [Telegram Cloud Storage](https://core.telegram.org/bots/webapps#cloudstorage). This package also handles stamping a Turnkey request with that api key. It is meant to be used with [`@turnkey/http`](https://www.npmjs.com/package/@turnkey/http)
```

Comment:
tiny nit for capitalization -- `Turnkey` and `API`

Reply:
updated